string tunnel
string led
string duty_cycle
---
string status